src/main/flight/pid.c

```diff
@@ -990,6 +991,22 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
             // Only increase ITerm if output is not saturated
             pidData[axis].I = ITermNew;
         }
+        #if defined(USE_TBH)
+            if (pidProfile->tbh_enabled){
+                // Non-linear Take-Back Half algorithm
+                // When the error term crosses through zero, this algorithm causes
+                // the I term to non-linearly jump to half-way between the current
+                // accumulated error and the accumulated error at the last zero
+                // crossing.
+                const float thisSign = copysign(1.0f, itermErrorRate); //-1 or 1
+                if (thisSign != pidData[axis].errorSignPrevTBH) { 
+                    const float lastCrossIValue = pidData[axis].IPrevTBH;
```

Comment:
TBH is meant to be an acronym.  Given that, is "Tbh" still preferable?  Makes it less readable IMHO, but I have no problem updating it to conform to the styleguide

Reply:
Generally, acronyms are used in camelCase by only capitalising the first letter - this allows for them to be used at the start of lower cased names without looking ugly, i.e. `tbhLimit` instead of `tBHLimit`.